train.py

```diff
@@ -330,8 +329,25 @@ def buildCallbacks(params, model, dataset):
 def main():
     args = parse_args()
     print(args)
-    parameters = load_parameters()
-    if args.config is not None:
+    # load the default config parameters
+    # load the user config and overwrite any defaults
+    if args.config.endswith('.yml'):
+        with open('configs/default-config-BiRNN.yml') as file: #FIXME make this a user option (maybe depend on model type and level?)
+            parameters = yaml.load(file, Loader=yaml.FullLoader)
+        with open(args.config.strip()) as file:
+            user_parameters = yaml.load(file, Loader=yaml.FullLoader)
+        for key in user_parameters:
+            parameters[key] = user_parameters[key]
```

Comment:
I quite like the clarity of this particular bit of python, but you can also replace it with: `parameters.update(user_parameters)`